src/frontend/src/app/app.module.ts

```diff
@@ -16,7 +16,7 @@ import { HomeComponent } from './home/index';
 import { LoginComponent } from './login/index';
 import { FooterComponent } from './footer/index';
 
-import { KeysPipe, FilterPipe } from './pipes/index';
+import { KeysPipe, FilterPipe, MomentPipe } from './pipes/index';
```

Comment:
Change `MomentPipe` pipe name to more expressive name towards the functionality  of that i.e. `MomentDatePipe`